--- origsrc/namebench-1.2/setup.py	2010-02-28 15:52:41.000000000 -0600
+++ src/namebench-1.2/setup.py	2010-05-21 19:58:48.857007800 -0500
@@ -113,18 +113,18 @@ setup(name='namebench',
       license='Apache 2.0',
       scripts=['namebench.py'],
       data_files=[
-          ('namebench/config',
+          ('share/namebench/config',
            ['config/namebench.cfg',
             'config/hostname_reference.cfg',
             'config/data_sources.cfg']
           ),
-          ('namebench/templates',
+          ('share/namebench/templates',
            ['templates/ascii.tmpl',
                 'templates/html.tmpl',
                 'templates/style.css'
            ]
           ),
-          ('namebench/data', ['data/alexa-top-10000-global.txt'])
+          ('share/namebench/data', ['data/alexa-top-10000-global.txt'])
       ],
 
       # py2exe specific garbarge below.
