--- origsrc/namebench-1.3.1/libnamebench/geoip.py	2010-06-06 11:19:28.000000000 -0500
+++ src/namebench-1.3.1/libnamebench/geoip.py	2010-12-24 01:15:24.398172700 -0600
@@ -19,7 +19,7 @@ import tempfile
 
 # external dependencies (from nb_third_party)
 import httplib2
-import simplejson
+import json as simplejson
 
 import util
 
--- origsrc/namebench-1.3.1/libnamebench/reporter.py	2010-06-06 11:19:28.000000000 -0500
+++ src/namebench-1.3.1/libnamebench/reporter.py	2010-12-24 01:15:30.768181500 -0600
@@ -22,7 +22,7 @@ import platform
 
 # external dependencies (from third_party)
 import jinja2
-import simplejson
+import json as simplejson
 
 import addr_util
 import charts
--- origsrc/namebench-1.3.1/libnamebench/site_connector.py	2010-06-06 11:19:28.000000000 -0500
+++ src/namebench-1.3.1/libnamebench/site_connector.py	2010-12-24 01:15:35.598188100 -0600
@@ -26,7 +26,7 @@ import zlib
 
 # third_party
 import httplib2
-import simplejson
+import json as simplejson
 
 import util
 
